### PR TITLE
translation to Dutch

### PR DESCRIPTION
**Pull request by Joppla**
Please check if the Dutch word [agenda's] needs an escape character (\ or /) before  the  [ ' ]. I wasn't sure.

### CODE REVIEW

**Comment by forrestguice** [comment]
Thanks. I should have time to review/merge this in the next day or so.

btw, don't pay any attention to that failing check; the automated build seems to have been broken recently. Its something I'll have to fix.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -236,7 +236,7 @@
         <item><![CDATA[<a href=https://github.com/FTno>FTno</a>]]></item>
         <item><![CDATA[<a href=https://github.com/pggdt>ft42</a>]]></item>
         <item><![CDATA[<a href=https://github.com/NetoSilva>Neto Silva</a>|<a href=https://github.com/naoliv>Nelson&#160;A.&#160;de&#160;Oliveira</a>|<a href=https://github.com/efraletti>Enrico S. B. Fraletti</a>]]></item>
-        <item><![CDATA[<a href=optionalUrlHere>TODO</a>]]></item>
+        <item><![CDATA[<a href=https://github.com/Joppla>Joppla</a>]]></item>
```
Comment (forrestguice) [inline app/src/main/res/values/strings.xml]: @Joppla The attribution here is whatever you'd like. Its displayed in the About dialog and the readme.

**Comment by forrestguice** [comment]
This looks good, thank you.
I'm going to tag v0.5.1 this weekend. It should hopefully be on F-Droid the week after.

btw the `'` character does need to be escaped with a `\` so I made that change. I also added some attribution (let me know if I should change that). If you aren't opposed, I'm going to migrate some of these values over to the main app as a starting point for future translation.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Joppla) [reply app/src/main/res/values/strings.xml]: It is oke

**Comment by Joppla** [comment]
I m glad that you are happy with the result.

**Pull request merged**